explorer/app2/index.html

```diff
@@ -60,19 +60,13 @@
             <!-- Page 0 -->
             <div id="page-0" class="page-0">
                 <div class="row">
-                    <div class="col has-text-centered">
+                    <!-- <div class="col has-text-centered">
```

Comment:
We can remove this piece of code, if we're not actually using it and it's just commented out. 😃

Reply:
It's only a temporary commenting-out.  When the tests resume, this code will have to be un-commented.